helm-chart/templates/nginx-mesh-api.yaml

```diff
@@ -176,19 +176,40 @@ spec:
 apiVersion: admissionregistration.k8s.io/v1
 kind: MutatingWebhookConfiguration
 metadata:
-  name: sidecar-injector-webhook-cfg.internal.builtin.nsm.nginx
+  name: sidecar-injector-webhook-ns.internal.builtin.nsm.nginx
   labels:
     app.kubernetes.io/part-of: nginx-service-mesh
     spiffe.io/webhook: "true"
 webhooks:
 - name: nginx-mesh-api.sidecar.injector
   namespaceSelector:
-    matchExpressions:
-    - key: kubernetes.io/metadata.name
-      operator: NotIn
-      values:
-      - kube-system
-      - {{ .Release.Namespace }}
```

Comment:
We should leave these hardcoded because we never want to inject here, even if a user specifies it.